core/src/mindustry/content/Blocks.java

```diff
@@ -536,6 +536,27 @@ public void drawBase(Tile tile){}
             consumes.items(new ItemStack(Items.coal, 4), new ItemStack(Items.sand, 6), new ItemStack(Items.pyratite, 1));
             consumes.power(4f);
         }};
+        
+        plastaniumPolymerizer = new AttributeCrafter("plastanium-polymerizer"){{
+            requirements(Category.crafting, with(Items.titanium, 100, Items.graphite, 45, Items.plastanium, 60, Items.lead, 60, Items.silicon, 80));
+            craftEffect = Fx.formsmoke;
+            health = 450;
+            updateEffect = Fx.plasticburn;
+            outputItem = new ItemStack(Items.plastanium, 4);
+            craftTime = 70f;
+            size = 3;
+            hasPower = hasLiquids = true;
+            drawer = new DrawGlow();
+            itemCapacity = 25;
+            boostScale = 0.20f;
+            liquidCapacity = 135f;
+            attribute = Attribute.water;
+
+            consumes.items(new ItemStack(Items.titanium, 4), new ItemStack(Items.graphite, 2));
+            consumes.liquid(Liquids.oil, 0.70f);
+            consumes.power(4.5f);
+            
+        }};
```

Comment:
```suggestion
            consumes.power(4.5f);
        }};
```

Reply:
please no. after that I get errors. I will do it myself in order to avoid numerous and difficult mistakes.

Reply:
_the art of pulling, then pushing_

Reply:
( ͡° ͜ʖ ͡°)